k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml

```diff
@@ -2,122 +2,152 @@
   dmap:
     "sha256:10b30e54a6367e69c52c22998f6f154475b7c94b7a674324f1b96b549048c50e": ["v1.19.0-rc.2"]
     "sha256:288138b199735aa767b042a2bbdc211fb852bf2e5e8368ab45e42fa1ddb0373f": ["v1.19.0-rc.3"]
+    "sha256:4fccdf9bfd019a8871729a23d9be4055466b98f857fd811b2141f90bc60c86cc": ["v1.19.0-rc.4"]
 - name: conformance-amd64
   dmap:
     "sha256:8c053fbfe74da394cf2c9ace242a060a27f62cb701b0d76f1377e14c0cc7ce4e": ["v1.19.0-rc.2"]
     "sha256:c84f41a851478e60c143e9dd98fe9ab3caca08ed598652a8e308d47855c3ab43": ["v1.19.0-rc.3"]
+    "sha256:ea2b4c82b7b524aa1dc86da3d75ac142bc6ccd188e57df2f06cc220052c1eb9c": ["v1.19.0-rc.4"]
 - name: conformance-arm
   dmap:
     "sha256:aac6803b665bc435193caa03ced25d7dfc4dd06f5e671fffcf8866abcefbdc15": ["v1.19.0-rc.3"]
+    "sha256:b92cc0d5f677e10243fe886d049647fc3b2b6fed1f69ac3d4fd2ece6a36f207e": ["v1.19.0-rc.4"]
```

Comment:
Not a big deal at all, but I wonder if the generation tool could be modified to keep the images in semver order

Reply:
i will check the tool and see where we can make that